src/Tool/Newsletter.php

```diff
@@ -204,7 +204,12 @@ public static function sendNewsletterDocumentBasedMail(Mail $mail, SendingParamC
 
     protected static function obfuscateEmail(string $email): string
     {
-        return substr_replace($email, '.xxx', strrpos($email, '.'));
+        try {
+            return substr_replace($email, '.xxx', strrpos($email, '.'));
+        } catch (\TypeError) {
+            // if the email address is too broken to be obfuscated, just return it
+            return $email;
+        }
```

Comment:
I would prefer a if instead of a try/catch
```suggestion
        $offset = strrpos($email, '.');
        if ($offset !== false) {
            return substr_replace($email, '.xxx', $offset);
        }
        // if the email address is too broken to be obfuscated, just return it
        return $email;
```